bitcoin-core-dev-tech/2018-03-05-bellare-neven.md

```diff
@@ -1,8 +1,10 @@
 ---
-title: Bellare Neven (2018-03-05)
+title: Bellare Neven 
 transcript_by: Bryan Bishop
 categories: ['core-dev-tech']
 tags: ['multisig']
+speakers: ['Peter Wuille']
```

Comment:
How did you determine the speaker here? Is it just personal knowledge? Cause it doesn't seem to be mentioned in the transcript. If this is indeed true, can you also include it in the spanish translation?

Reply:
I checked the `bitcoin-core-dev` link in the article and `Peter Wille (SIPA)` was mentioned in the beginning, so I assumed he was the speaker. However, that might be wrong so I will take it out.